sound/soc/sof/amd/acp.h

```diff
@@ -174,6 +174,7 @@ struct sof_amd_acp_desc {
 	const char *name;
 	unsigned int host_bridge_id;
 	u32 pgfsm_base;
+	u32 ext_intr_enb;
```

Comment:
actually there is nothing in this PR that sets this member, so is this even used?

Reply:
@plbossart : For Phoenix platform, it will be set.  We are going to push a separate commit for it as we are going to add SOF support for Phoneix platform.